controllers/cleanup_controller.go

```diff
@@ -74,6 +82,28 @@ func (r *CleanupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	return ctrl.Result{}, nil
 }
 
+func (r *CleanupReconciler) isOcsOperatorSubAndCSVAt419(ctx context.Context) error {
+	subscriptionList := &opv1a1.SubscriptionList{}
+	if err := r.List(ctx, subscriptionList, client.InNamespace(r.OperatorNamespace)); err != nil {
+		return fmt.Errorf("failed to list subscriptions: %v", err)
+	}
+	ocsOperatorSubscription := util.Find(subscriptionList.Items, func(sub *opv1a1.Subscription) bool {
+		return sub.Spec.Package == "ocs-operator"
+	})
+	if ocsOperatorSubscription == nil {
+		return fmt.Errorf("no subscription exists with package 'ocs-operator'")
+	} else if strings.HasSuffix(ocsOperatorSubscription.Spec.Channel, "4.18") {
+		return fmt.Errorf("subscription of 'ocs-operator' still points to '4.18'")
+	} else if !strings.HasSuffix(ocsOperatorSubscription.Spec.Channel, "4.19") {
```

Comment:
Why are we using a suffix check here? what will happen with dev/test builds?

Reply:
The channel is usually prefixed w/ `stable-`, this doesn't effect dev testing as upgrades is validated w/ d/s images.